talon/quotations.py

```diff
@@ -72,17 +72,21 @@
     ))
 # Special case for languages where text is translated like this: 'on {date} wrote {somebody}:'
 RE_ON_DATE_WROTE_SMB = re.compile(
-    u'(-*[>]?[ ]?({0})[ ].*(.*\n){{0,2}}.*({1})[ ]*.*:)'.format(
+    u'(-*[>]?[ ]?({0})[ ].*(.*\n){{0,2}}.*({1})[ ]*.*)'.format(
```

Comment:
Looking at the pattern above `RE_ON_DATE_SMB_WROTE` I'd actually merge it with this one - `RE_ON_DATE_WROTE_SMB`. They probably started like different patterns but eventually evolved into something very similar. I'd also leave the colon `:` - the sample in the test below has it but on a 3rd line (are they (email clients) freaking kidding me!) . So, I'd just allowed one more `\n` in the splitter.